### PR TITLE
Admin Dashboard (Campaign List)

### PR DESCRIPTION
**Pull request by coderberry**
# Type of PR (feature, enhancement, bug fix, etc.)

Enhancement

Fixes #(issue)

It's difficult to have clarity on how campaigns are currently performing. This gives me a better view of the overall health of all live campaigns## Checklist

### General

- [x] My code follows the style guidelines of this project & the tests are passing
- [x] I have performed a self-review of my own code
- [x] I have updated DB schema annotations or I have not made any DB schema changes

### CodeFund Core Team

- [x] I have added the appropriate milestone if applicable
- [x] I have requested a review from another developer(s)

## Screenshots

CodeFund___Ethical_Advertising

### CODE REVIEW

**Comment by andrewmcodes** [comment]
There is a startling lack of helpers in here. 😉 I'll push this to the finish line for you when I get a chance @coderberry

**Comment by andrewmcodes** [comment]
CodeFund | Ethical Advertising 2020-02-28 13-16-06

File: app/models/concerns/campaigns/budgetable.rb [hunk 1]
```diff
@@ -42,13 +42,13 @@ def selling_price
 
     # Returns a Money indicating how much budget has been spent
     def total_consumed_budget
-      gross_revenue start_date, Date.current
+      @total_consumed_budget ||= gross_revenue start_date, Date.current
```
Comment (hopsoft) [inline app/models/concerns/campaigns/budgetable.rb]: I'd throw parens on this now that it's on the right hand side of an assignment.

Comment (hopsoft) [reply app/models/concerns/campaigns/budgetable.rb]: I'd be wary of memoizing this unless you saw a big performance win from adding it.

**Pull request merged**